mainapp/helpers.py

```diff
@@ -1,3 +1,4 @@
+"""import modules. """
```

Comment:
Only document the functions & Classes if any :)

Reply:
It won't show 100% coverage without this line..it asks to add docstring for the module as well :/

Reply:
Oh, that's alright, ~90%+ coverage is good enough as long as methods & classes are documented